### DISCUSSION TITLE
Support Grafana Time variables in URL

### DISCUSSION THREAD

**Discussion by beirtipol** [grafana/grafana-infinity-datasource#485] [Ideas]
I'd like to be able to query for specific time ranges rather than having to load all the data and then filter in grafana. 

e.g. querying for builds in a certain timerange in teamcity, you can see from the Query inspector that the 'from' and '__to' dates are not getting resolved

`http://teamcity/app/rest/builds?locator=sinceDate:{__from:date:YYYYMMDD'T'HHmmss},untilDate:{__to:date:YYYYMMDD'T'HHmmss}&fields=build(id,number,agent,status,finishOnAgentDate,statistics(property(name,value)))`

```
###############
## URL
###############

http://teamcity/app/rest/builds?fields=build%28id%2Cnumber%2Cagent%2Cstatus%2CfinishOnAgentDate%2Cstatistics%28property%28name%2Cvalue%29%29%29&locator=sinceDate%3A%7B__from%3Adate%3AYYYYMMDD%27T%27HHmmss%7D%2CuntilDate%3A%7B__to%3Adate%3AYYYYMMDD%27T%27HHmmss%7D
```

**Comment by yesoreyeram** [comment]
Looks like your are missing `$` infront of the variable. `{__from` part should be `${__from`.
